examples/demo-atmos.d/atmos.d/sub/config2.yaml

```diff
@@ -0,0 +1,5 @@
+logs:
+  level: Warning
```

Comment:
I think for an example, this is not great, it's not showing how/why to create different configurations. For example, to keep `integrations` in one file, or `terraform` settings in another, and `helmfile` in another.